packages/react-instantsearch-hooks-web/src/ui/ShowMoreButton.tsx

```diff
@@ -2,13 +2,30 @@ import React from 'react';
 
 type ShowMoreButtonProps = React.ComponentProps<'button'> & {
   isShowingMore: boolean;
+  translations: ShowMoreButtonTranslations;
+};
+
+export type ShowMoreButtonTranslations = {
+  /**
+   * Alternative text for the show more button.
+   */
+  showMoreButtonText: string;
+  /**
+   * Alternative text for the show less button.
+   */
+  showLessButtonText: string;
 };
 
 export function ShowMoreButton({
   isShowingMore,
+  translations,
   ...props
 }: ShowMoreButtonProps) {
   return (
-    <button {...props}>{isShowingMore ? 'Show less' : 'Show more'}</button>
+    <button {...props}>
+      {isShowingMore
+        ? translations.showLessButtonText
+        : translations.showMoreButtonText}
```

Comment:
I agree, with the new wording it seems like there's a separate show less button, so I'd make this `showMoreButtonText(isShowingMore)`, probably object arguments for everything (even though it bloats bundlesize)